### PR TITLE
add check if the wf is running. if running reject the deletion

### PR DESCRIPTION
**Pull request by drekoumis**
When you call the API /api/delete/<workflow-id> the workflow is not running. If it's running a message should appear mentioning that you can't delete the current workflow. Change the status of the workflow from Running to error when a job error arrives.

### CODE REVIEW

File: panoptes/routes/api.py [hunk 1]
```diff
@@ -70,12 +70,14 @@ def get_job_of_workflow(workflow_id, job_id):
 
 @routes.route('/api/delete/<workflow_id>', methods=['GET'])
 def set_db_delete(workflow_id):
-    if(get_db_workflows_by_id(workflow_id)!=None):
+    if(get_db_workflows_by_id(workflow_id)==None):
```
Comment (fgypas) [inline panoptes/routes/api.py:73]: I think you need to check with `is` instead of `==` when you compare to None.

File: panoptes/routes/api.py [hunk 2]
```diff
@@ -70,12 +70,14 @@ def get_job_of_workflow(workflow_id, job_id):
...
+        return jsonify({'error': 404, 'msg': 'Unable to delete Workflow ' + workflow_id + 
+                        '. Please check if workflow ' + workflow_id + ' exists.'})
+    elif(get_db_workflows_by_status(workflow_id)=='Running'):
```
Comment (fgypas) [inline panoptes/routes/api.py:76]: Maybe the best is to call `get_db_workflows_by_status(workflow_id)` once before the if statements. Now you call the same thing twice.

**Review by fgypas — CHANGES_REQUESTED** [review]
Some minor changes. I did not change the functionality yet. I will check once more after you push.

File: panoptes/routes/api.py (hunk 2 above)
Comment (drekoumis) [reply panoptes/routes/api.py:76]: We have 2 deferment errors. I thing is better to check if wf is on database and after check if his status is Running.

File: panoptes/routes/api.py (hunk 1 above)
Comment (drekoumis) [reply panoptes/routes/api.py]: done

**Review by fgypas — APPROVED** [review]
Looks good to me!

**Pull request merged**